i18n/translations/fr-FR.json

```diff
@@ -225,6 +225,7 @@
     "Use camera's entropy to create a new mnemonic": "Utilisez l'entropie de la caméra pour créer un nouveau mnémonique",
     "Used: ": "Utilisé: ",
     "Value %s out of range: [%s, %s]": "Valeur% s hors de portée: [% s,% s]",
+    "Value must be multiple of %s": "La valeur doit être multiple de% s",
```

Comment:
"La valeur doit être un multiple de %s"

No big deal on the missing "un", but the space is on the wrong side of the % above and won't replace, i think.

Reply:
oh I missed the "un". If I get another erro I'll put this together, otherwise let's leave it for next release

Reply:
Added this fix to integrated_changes after I merged changes from main